### PR TITLE
2022-05-27-rct_binary_classifier_biobert_en_3_0

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

**Comment by Ahmetemintek** [comment]
model was re-loaded with jsl v3.4.2

File: docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+result = biobert_clf_pipeline.fit(data).transform(data)
+```
+```scala
+val documenter = new DocumentAssembler() 
+    .setInputCol("text") 
+    .setOutputCol("document")
+
+val bert_sent = new BertSentenceEmbeddings.pretrained("sent_biobert_pubmed_base_cased", "en")
```
Comment (muhammetsnts) [inline docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md:66]: Please check these .pretrained lines, there shouldn't be `new` here.

File: docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    .setInputCols("document")
+    .setOutputCol("sentence_embeddings")
+
+val classifier_dl = new ClassifierDLModel.pretrained("rct_binary_classifier_biobert", "en", "clinical/models")
```
Comment (muhammetsnts) [inline docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md:70]: also this one

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md (hunk 2 above)
Comment (Ahmetemintek) [reply docs/_posts/Ahmetemintek/2022-05-27-rct_binary_classifier_biobert_en_3_0.md]: I've just fixed it, thanks for the feedback!

**Review by muhammetsnts — APPROVED** [review]

**Pull request merged**